src/model/Request.ts

```diff
@@ -0,0 +1,12 @@
+export default interface Request {
+  id: number
+  major: string
+}
+
+export function getRequestJson(requestFormData: any): Request {
+  return {
+    id: 0,  // TODO needs to be changed for each request.
```

Comment:
As of now, there's no need for the client side to keep track of the id as there will one and only one request from one client on the fly at any point of time. 

Does not affect merge.

Reply:
I agree, for the time being we can fix that value to always be 0 👍